### PR TITLE
Update apm.md

### PR DESCRIPTION
**Pull request by andrewardito**
### What does this PR do?
Update Node ECS on EC2 documentation

### CODE REVIEW

File: content/en/agent/amazon_ecs/apm.md [hunk 1]
```diff
@@ -96,14 +96,14 @@ For more examples of setting the Agent hostname in other languages, refer to the
 
 ```javascript
 const tracer = require('dd-trace');
+tracer.init();
```
Comment (bengl) [inline content/en/agent/amazon_ecs/apm.md:99]: The init can also be on the same line as the require, like:

`const tracer = require('dd-trace').init();`

**Review by bengl — CHANGES_REQUESTED** [review]

File: content/en/agent/amazon_ecs/apm.md [hunk 2]
```diff
@@ -96,14 +96,14 @@ For more examples of setting the Agent hostname in other languages, refer to the
 
 ```javascript
 const tracer = require('dd-trace');
```
Comment (andrewardito) [inline content/en/agent/amazon_ecs/apm.md:98]:
```suggestion
const tracer = require('dd-trace').init();
```

File: content/en/agent/amazon_ecs/apm.md [hunk 3]
```diff
@@ -95,15 +95,15 @@ For more examples of setting the Agent hostname in other languages, refer to the
 {{% tab "Node.js" %}}
 
 ```javascript
-const tracer = require('dd-trace');
+const tracer = require('dd-trace').init();
+tracer.init();
```
Comment (andrewardito) [inline content/en/agent/amazon_ecs/apm.md:99]:
```suggestion
```

File: content/en/agent/amazon_ecs/apm.md (hunk 1 above)
Comment (andrewardito) [reply content/en/agent/amazon_ecs/apm.md]: done!

**Review by sarina-dd — APPROVED** [review]

**Pull request merged**